content/software/plc-ide/tutorials/plc-programming-introduction/content.md

```diff
@@ -100,10 +104,32 @@ Inside the **Tool window** you can drag and drop variables, or click the **Inser
 
 ![Watch a variable](assets/watch.png)
 
+### Library Management
+
+The PLC IDE manages the library add-on and usage differently. To add the desired library, go to **Resources** tab and click the **Libraries** listed under the **Sketch**.
```

Comment:
```suggestion
The PLC IDE manages the library add-on and usage differently from the traditional Arduino IDE. To add the desired library, go to **Resources** tab and click the **Libraries** listed under the **Sketch**.
```

Reply:
just to add context... "differently from what..." should be question of the user reading that line.